src/backend/distributed/commands/alter_table.c

```diff
@@ -1183,6 +1204,75 @@ ReplaceTable(Oid sourceId, Oid targetId, List *justBeforeDropCommands,
 }
 
 
+/*
+ * HasAnyGeneratedStoredColumns decides if relation has any columns that we
+ * might need to copy the data of when replacing table.
+ */
+static bool
+HasAnyGeneratedStoredColumns(Oid relationId)
+{
+	return list_length(GetNonGeneratedStoredColumnNameList(relationId)) > 0;
+}
+
+
+/*
+ * BuildInsertColumnStringForReplaceTable is a helper function that returns
+ * comma seperated columns that we can insert into for ReplaceTable.
+ */
+static char *
+BuildInsertColumnStringForReplaceTable(Oid relationId)
```

Comment:
This functions name is obsolete with the latest changes. In fact, we can get rid of this function and inline the two lines just before `appendStringInfo ` as `GetNonGeneratedStoredColumnNameList ` makes it much easier to follow.
(as I noted in the other comment, moving this comment above sounds better to me)